openedx_external_enrollments/models.py

```diff
@@ -38,3 +40,26 @@ class Meta(object):
         Model meta class.
         """
         app_label = "openedx_external_enrollments"
+
+
+class OtherCourseSettings(models.Model):
+    """
+    Model to persist other course settings.
+    """
+
+    course_overview = models.OneToOneField(
+        CourseOverview,
+        on_delete=models.CASCADE,
+        primary_key=True,
+    )
+    external_course_id = models.CharField(max_length=50, null=False, blank=True)
+    external_platform = models.CharField(max_length=20, null=False, blank=True)
+
+    class Meta(object):
+        """
+        Model meta class.
+        """
+        app_label = "openedx_external_enrollments"
+
+    def __str__(self):
+        return str(course_overview.id)
```

Comment:
Is this required? @ivanvgh

Reply:
It is required of the representation when making queries.